### DISCUSSION TITLE
Unable to login via Zen Browser

### DISCUSSION THREAD

**Discussion by tismofied** [FreshRSS/FreshRSS#6921] [Q&A] [answered]
### Describe the bug

I moved browsers from Vivaldi which is based on chrome to zen browser which is in turn based on Firefox. I am able to login in Vivaldi with password and username ( not using proxy or anything) but keep getting "login invalid" using Zen browser.
How can I troubleshoot why?

### FreshRSS version

1.24.3

### Environment information

- Database version: [e.g. Mysql 5.7, SQLite] SQlite
- PHP version: [e.g. PHP 8.1]  PHP version (8.3.12)
- Installation type: [e.g. Docker, Docker image source, git, Yunohost] Docker, linuxserver.io
-Web server type: [e.g. Apache, nginx]
- Device: [e.g. iPhone13]
- OS: [e.g. Ubuntu 22.04, Win10, MacOS14] debian 12
- Browser: [e.g. Firefox 124] Zen Browser

**Comment by Frenzie** [comment, marked as answer]
It sounds like maybe it's blocking Javascript by default? If you're on HTTPS it should be fairly safe to choose an authentication method without Javascript, but everything is quite hampered without regardless.

**Comment by Alkarex** [comment]
Without JavaScript, the HTTP Basic Auth method can be used (or OpenID Connect) instead of the form-based login
